File: TasKagitMakas/TasKagitMakas.java
package taskagitmakas;

import java.util.Random;
import java.util.Scanner;

public class TasKagitMakas {

    public static void main(String[] args) {
        Random rastgele = new Random();
        int bilgisayar = 1 + rastgele.nextInt(3);
        System.out.println("1) ♖");
        System.out.println("2) #");
        System.out.println("3) ✂");
        Scanner reader = new Scanner(System.in);
        System.out.print("Bir icon seçin: ");
        int oyuncu = reader.nextInt();
        for (;;) {
            if (oyuncu == 1 && bilgisayar == 2) {
                System.out.println("Kağıt Taşı Sarar");
                System.out.println("KAYBETTİN");
                break;

            } else if (oyuncu == 1 && bilgisayar == 3) {
                System.out.println("Taş Makası Kırar");
                System.out.println("KAZANDIN");
                break;

            } else if (oyuncu == 2 && bilgisayar == 1) {
                System.out.println("Kağıt Taşı Sarar");
                System.out.println("KAZANDIN");
                break;

            } else if (oyuncu == 2 && bilgisayar == 3) {
                System.out.println("Makas Kağıdı Keser");
                System.out.println("KAYBETTİN");
                break;

            } else if (oyuncu == 3 && bilgisayar == 1) {
                System.out.println("Taş Makası Kırar");
                System.out.println("KAYBETTİN");
                break;

            } else if (oyuncu == 3 && bilgisayar == 2) {
                System.out.println("Makas Kağıdı Keser");
                System.out.println("KAZANDIN");
                break;

            } else {
                System.out.println("BERABERE");
                break;
            }
        }
    }
}

File: TasKagitMakas/TasKagitMakas.py
import random


def tas_kagit_makas():

    envanter = ['Taş', 'Kağıt', 'Makas']
    bilgisayar = (envanter[random.randint(0, 2)]).upper()
    oyuncu = (input("(Taş)\n(Kağıt)\n(Makas)\nBirini seçin : ")).upper()

    if oyuncu == "TAŞ":
        if bilgisayar == "TAŞ":
            print("Berabere")
        elif bilgisayar == "KAĞIT":
            print("Kağıt taşı sarar\nKaybettin")
        elif bilgisayar == "MAKAS":
            print("Taş makası kırar\nKazandın")
    elif oyuncu == "KAĞIT":
        if bilgisayar == "TAŞ":
            print("Kağıt taşı sarar\nKazandın")
        elif bilgisayar == "KAĞIT":
            print("Berabere")
        elif bilgisayar == "MAKAS":
            print("Makas kağıdı keser\nKaybettin")
    elif oyuncu == "MAKAS":
        if bilgisayar == "TAŞ":
            print("Taş makası kırar\nKaybettin")
        elif bilgisayar == "KAĞIT":
            print("Makas kağıdı keser\nKazandın")
        elif bilgisayar == "MAKAS":
            print("Berabere")
    else:
        print("Geçerli bir sayı girmediniz")


tas_kagit_makas()